src/text.h

```diff
@@ -316,7 +316,7 @@
 #define  w_CONTINUITY        "CONTINUITY"
 #define  w_FLOWSTATS         "FLOWSTATS"
 #define  w_CONTROLS          "CONTROL"
-#define  w_NODESTATS         "NODESTATS"
+#define  w_NODESTATS         "NODSTATS"
```

Comment:
Why doing those changes?

Reply:
That was a temporal change I did in order to see its effect on the results. Not important. I changed it back to its original form in a subsequent commit.